### DISCUSSION TITLE
Feedback // Beta// Private profile -> More custom

### DISCUSSION THREAD

**Discussion by QuentinHuber** [community/community#17209] [Other Feature Feedback, Questions, & Ideas]
### Feedback // Beta // Private profile

The idea of putting your profile in private is really cool, the problem is that there is no in-between.
We should instead try to propose things to be put in private via checkboxes, _example_, like putting your activity in private and leaving the projects where we put a star.

Transform Private Profile into Personalized/Custom Private Profile

Have a good day 😉

**Comment by asathkum** [comment]
> We should instead try to propose things to be put in private via checkboxes, _example_, like putting your activity in private and leaving the projects where we put a star.

More control over what is visible (and crawlable) on your profile sounds like a great idea! I for one would like to be able to pin repositories while hiding activity.
